assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -21,7 +21,9 @@ angular.module('openshiftConsole')
       var containers = dcContainers(dc);
       if (containers) {
         angular.forEach(containers, function(container) {
-          images.push(container.image);
+          if (container.image) {
```

Comment:
@jwforres I'm not sure, but it was switched to optional in `types.go`.

@liggitt This is the field whose validator still requires it, right?